components/hero/hero.css

```diff
@@ -0,0 +1,65 @@
+.ampstart-hero-content {
+  padding: calc(var(--space-4) + var(--space-1)) var(--narrow-page-padding);
+}
+
+.ampstart-hero-body {
+  padding-bottom: var(--space-3);
+  font-family: var(--font-family-secondary);
+  width: 100%;
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.ampstart-hero-title {
+  font-weight: var(--bold-font-weight);
+}
+
+@media (--breakpoint-md) {
+  .ampstart-hero {
+    position: relative;
+    color: var(--color-white);
+    text-align: left;
+  }
+
+  .ampstart-hero-title {
+    font-size: var(--h1);
+    color: var(--color-white);
+    font-weight: var(--bold-font-weight);
+  }
+
+  .ampstart-hero-body {
+    max-width: none;
+  }
+
+  .ampstart-hero-content {
+    position: absolute;
+    top: 0;
+    left: var(--space-2);
+    width: 45%;
+    height: 100%;
+  }
+
+  .ampstart-hero .ampstart-btn {
+    background-color: var(--color-white);
+    border: none;
+    color: var(--color-font-title);
+    font-size: var(--h5);
+  }
+
+  .ampstart-hero-theme-secondary .ampstart-hero-title,
+  .ampstart-hero-theme-secondary .ampstart-hero-body,
+  .ampstart-hero-theme-secondary .ampstart-hero .ampstart-btn {
+    color: var(--color-secondary);
+  }
+
+  .ampstart-hero-theme-secondary .ampstart-btn {
+    color: var(--color-white);
+    background-color: var(--color-secondary);
+  }
+}
+
+@media (--breakpoint-lg) {
+  .ampstart-hero-content {
+    width: 30%;
```

Comment:
Simplified design by removing 30% & 45% widths and setting a fixed width of 500px to parent container.